src/main/java/com/aws/greengrass/clientdevices/auth/PermissionEvaluationUtils.java

```diff
@@ -119,6 +130,7 @@ private boolean compareOperation(Operation requestOperation, String policyOperat
         if (requestOperation.getOperationStr().equals(policyOperation)) {
             return true;
         }
+        // TODO
```

Comment:
todo?

Reply:
yeah to replace string formats, will take care of those now